.github/labels.yml

```diff
@@ -91,44 +98,38 @@
 - name: kind/tracking-issue
   color: fcf0b5
   description: "Kind: Tracking Issue"
+- name: kind/epic
+  color: 3e4b9e
+  description: "Kind: EPIC"
 - name: kind/question
   color: fcf0b5
   description: "Kind: Question"
-- name: kind/enhancement
```

Comment:
Maybe we should leave a tag for "Feature" issues?

Reply:
There is a `kind/feature` a few lines above

Reply:
My bad! Didn't notice the lines collapsed.